### ISSUE TITLE
maximum recursion depth exceeded in ThreadSafeSwiftclientStorage get_connection

### ISSUE THREAD

**Issue by crucialfelix** [django-cumulus/django-cumulus#191]
Perhaps I've missed some login or initialization requirement or something, but the way this code is written it will blow up every single time.

If connection is not set then the method calls itself.

```python
django_cumulus-1.0.19-py2.7.egg/cumulus/storage.pyc in _get_connection(self)
    245     def _get_connection(self):
    246         if not hasattr(self.local_cache, "connection"):
--> 247             connection = self._get_connection()
    248             self.local_cache.connection = connection
    249
```

https://github.com/django-cumulus/django-cumulus/blob/master/cumulus%2Fstorage.py#L254

**Comment by ferrix** [comment]
Yes, that is a bug.

**Comment by crucialfelix** [comment]
should be 

```python
super(ThreadSafeSwiftclientStorage, self)._get_connection()
```

**Comment by ferrix** [comment]
Fixed in cc224787c5
